File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webdriver import ActionChains
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())


with webdriver.Firefox(service=service) as driver:

    actions = ActionChains(driver)

    driver.get("https://v1.training-support.net/selenium/input-events")

    print("Page title is: ", driver.title)

    # Left click
    actions.click().perform()
    active_side = driver.find_element(By.CLASS_NAME, "active")
    print("Active side number after left click: ", active_side.text)

    # DOUBLE CLICK
    actions.double_click().perform()
    active_side = driver.find_element(By.CLASS_NAME, "active")
    print("Active side number after double click: ", active_side.text)

    # RIGHT CLICK
    actions.context_click().perform()
    active_side = driver.find_element(By.CLASS_NAME, "active")
    print("Active side number after right click: ", active_side.text)

File: Selenium/Activities/Activity5.java
package activities;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity5 {
    public static void main(String[] args) {

        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        Actions builder = new Actions(driver);


        driver.get("https://v1.training-support.net/selenium/input-events");

        System.out.println("Home page title: " + driver.getTitle());

        // left click
        builder.click().pause(1000).build().perform();
        // front side text
        String frontText = driver.findElement(By.className("active")).getText();
        System.out.println(frontText);

        // left click
        builder.doubleClick().pause(1000).build().perform();
        // front side text
        frontText = driver.findElement(By.className("active")).getText();
        System.out.println(frontText);


        builder.contextClick().pause(1000).build().perform();

        frontText = driver.findElement(By.className("active")).getText();
        System.out.println(frontText);


        driver.close();
    }
}